README.md

```diff
@@ -115,18 +117,19 @@ and will make the action exit with error otherwise.
 The following version file formats are supported:
 
 - `.tool-versions`, as specified by [asdf: Configuration](https://asdf-vm.com/manage/configuration.html)
+- `.mise.toml`, as specified by [mise: Configuration](https://mise.jdx.dev/configuration.html)
 
 Supported version elements are the same as the ones defined for the YML portion of the action,
 with the following correspondence.
 
-#### `.tool-versions` format
+#### Mapping between YML and version file formats
```

Comment:
This should be "Mapping between YML and `version-file-type` formats", where we list `asdf` and `mise` instead of the file names (which can be chosen by the consumer).